less/v2/npc.less

```
@@ -0,0 +1,610 @@
+      }
+    }
+
+    /* AC, HD, HP, & Death Saves */
+    .vitals {
+      display: flex;
+      margin-top: -40px;
+      padding: 0 12px 0 6px;
+      transform: scale(1.1);
+
+      .right {
+        padding-top: 6px;
+        margin-left: -10px;
+        flex: 1;
+      }
+
+      .ac-badge {
+        filter: unset;
+        margin: unset;
+        position: relative;
+        width: 52px;
+        height: 52px;
+        line-height: 48px;
+        font-size: var(--font-size-18);
+        z-index: 1;
+      }
+
+      .meter:is(.hit-points, .hit-dice) .label { padding-inline-start: 12px; }
+
+      .meter.hit-points {
+        block-size: 20px;
+        font-size: var(--font-size-12);
+        .bonus { font-size: var(--font-size-10); }
+      }
+
+      .meter:is(.hit-dice, .split) {
+        block-size: 16px;
+        font-size: var(--font-size-10);
+        margin-top: -1px;
+      }
+
+      .tmp { width: 35px; }
+
+      .meter.split > div {
+        &.temp input { color: var(--dnd5e-color-blue); }
+        &.tempmax input { color: var(--dnd5e-color-purple); }
+        input::placeholder { text-transform: uppercase; }
+        &:last-child { border-inline-start: 1px dashed var(--dnd5e-color-gold); }
+      }
+
+      /* Death Saves */
+      .death-saves {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        margin-top: -8px;
+        padding-right: 16px;
+
+        > button {
+          width: 28px;
+          height: 28px;
+          --icon-fill: var(--dnd5e-color-gold);
+          --icon-size: 20px;
+          filter: drop-shadow(0 0 1px black);
+
+          &:hover {
+            box-shadow: none;
+            filter: drop-shadow(0 0 4px var(--color-shadow-primary));
+          }
+        }
+
+        .pips {
+          padding: 0;
+          gap: 3px;
+
+          .pip {
+            box-shadow: none;
+            width: 10px;
+            height: 10px;
+            font-size: var(--font-size-12);
+            border-width: 1px;
+          }
+        }
+      }
+    }
+  }
+
+  /* ---------------------------------- */
+  /*  Sheet Body                        */
+  /* ---------------------------------- */
+
+  .sheet-body {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    overflow: hidden auto;
+    scrollbar-color: var(--dnd5e-color-gold) transparent;
+
+    .main-content {
+      --sidebar-padding: 8px;
+      --sidebar-size: calc(var(--dnd5e-sheet-sidebar-width) + var(--sidebar-padding));
+      padding: 0 0 0 var(--sidebar-padding);
+      display: flex;
+      position: absolute;
+      width: 100%;
+      min-height: 100%;
+
+      /* Sidebar Collapser */
+      .sidebar-collapser {
+        position: absolute;
+        inset: 0 auto 0 var(--sidebar-size);
+        width: 16px;
+        padding: 0;
+        color: var(--dnd5e-color-gold);
+        transition: left 450ms ease;
+
+        &:hover, &:focus {
```

Might want to use `focus-visible` for this otherwise it stays highlighted after you click it and move your mouse away.

```suggestion
        &:hover, &:focus-visible {
```